packages/bot-web-ui/src/components/dashboard/dashboard-component/__tests__/info-panel.spec.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { mockStore, StoreProvider } from '@deriv/stores';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { render, screen } from '@testing-library/react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import userEvent from '@testing-library/user-event';
+import { DBOT_TABS } from 'Constants/bot-contents';
+import { mock_ws } from 'Utils/mock';
+import RootStore from 'Stores/index';
+import { DBotStoreProvider, mockDBotStore } from 'Stores/useDBotStore';
+import InfoPanel from '../info-panel';
+
+jest.mock('@deriv/bot-skeleton/src/scratch/blockly', () => jest.fn());
+jest.mock('@deriv/bot-skeleton/src/scratch/dbot', () => ({
+    saveRecentWorkspace: jest.fn(),
+    unHighlightAllBlocks: jest.fn(),
+}));
+jest.mock('@deriv/bot-skeleton/src/scratch/hooks/block_svg', () => jest.fn());
```

Comment:
@farabi-deriv test cases will be passed without all these mocks:
```suggestion
jest.mock('@deriv/bot-skeleton/src/scratch/dbot', () => jest.fn());
```
<img width="1060" alt="Screenshot 2023-12-18 at 10 17 28" src="https://github.com/binary-com/deriv-app/assets/103181650/893dc777-439f-42e2-85aa-7f934c92a6fd">

Reply:
updated @maryia-matskevich-deriv